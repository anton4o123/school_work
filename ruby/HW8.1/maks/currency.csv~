EUR	Euro,0.7244281138,1.3803992156
GBP	British Pound,0.6175314631,1.6193506887
INR	Indian Rupee,61.4616075384,0.0162703196
AUD	Australian Dollar,1.0403201085,0.9612425943
CAD	Canadian Dollar,1.0425283369,0.9592065410
AED	Emirati Dirham,3.6729532988,0.2722604723
MYR	Malaysian Ringgit,3.1595400635,0.3165017629
CHF	Swiss Franc,0.8920590797,1.1210019860
CNY	Chinese Yuan Renminbi,6.0821113774,0.1644165879
THB	Thai Baht,31.1547980152,0.0320977847
SAR	Saudi Arabian Riyal,3.7504149303,0.2666371638
NZD	New Zealand Dollar,1.1970984604,0.8353531752
JPY	Japanese Yen,97.3397512488,0.0102732952
SGD	Singapore Dollar,1.2365495679,0.8087019121
PHP	Philippine Peso,43.0753258436,0.0232151465
TRY	Turkish Lira,1.9788004345,0.5053566709
HKD	Hong Kong Dollar,7.7535660931,0.1289729123
IDR	Indonesian Rupiah,11136.3880449002,0.0000897957
ZAR	South African Rand,9.7653020351,0.1024033866
MXN	Mexican Peso,12.9723007738,0.0770873276
SEK	Swedish Krona,6.3567512964,0.1573130603
BRL	Brazilian Real,2.2023315417,0.4540642410
HUF	Hungarian Forint,211.7327379537,0.0047229352
PKR	Pakistani Rupee	106.5198997274,0.0093879172
QAR	Qatari Riyal,3.6414910301,0.2746127868
OMR	Omani Rial,0.3850983907,2.5967389742
KWD	Kuwaiti Dinar,0.2818497782,3.5479893100
DKK	Danish Krone,5.4035159173,0.1850646903
NOK	Norwegian Krone,5.8954319921,0.1696228540
RUB	Russian Ruble,31.7087050068,0.0315370811
EGP	Egyptian Pound,6.8895413235,0.1451475437
KRW	South Korean Won,1061.9398554851,0.0009416729
PLN	Polish Zloty,3.0308085617,0.3299449568
COP	Colombian Peso,1886.4854964323,0.0005300862
CZK	Czech Koruna,18.6246581652,0.0536922606
ILS	Israeli Shekel,3.5232287897,0.2838305599
IQD	Iraqi Dinar,1164.9993639110,0.0008583696
NGN	Nigerian Naira,158.9431898628,0.0062915561
MAD	Moroccan Dirham,8.1478316688,0.1227320397
ARS	Argentine Peso,5.8668316912,0.1704497508
LKR	Sri Lankan Rupee,130.5498367474,0.0076599100
TWD	Taiwan New Dollar,29.4279884102,0.0339812557
BDT	Bangladeshi Taka,77.6958892489,0.0128706938
BHD	Bahraini Dinar,0.3770993287,2.6518212151
VND	Vietnamese Dong,21079.6141654522,0.0000474392
CLP	Chilean Peso,503.8494094885,0.0019847200
KES	Kenyan Shilling,84.7998365870,0.0117924756
TND	Tunisian Dinar,1.6254997531,0.6151954180
XOF	CFA Franc,475.1936922266,0.0021044050
JOD	Jordanian Dinar,0.7082515975,1.4119276306
GHS	Ghanaian Cedi,2.1874981803,0.4571432374
HRK	Croatian Kuna,5.5233237994,0.1810504030
BGN	Bulgarian Lev,1.4168770118,0.7057775598
RON	Romanian New Leu,3.2242432907,0.3101502926
PEN	Peruvian Nuevo Sol,2.7689985814,0.3611413912
DZD	Algerian Dinar,81.5007321051,0.0122698284
NPR	Nepalese Rupee,98.6774264532,0.0101340300
XAF	Central African CFA Franc BEAC,475.1936922266,0.0021044050
ISK	Icelandic Krona,120.0999231360,0.0083264000
UAH	Ukrainian Hryvna,8.1779993986,0.1222792949
FJD	Fijian Dollar,1.8141873076,0.5512110000
DOP	Dominican Peso,42.3079996555,0.0236361919
XPF	CFP Franc,86.4472689467,0.0115677454
MUR	Mauritian Rupee,30.4460556983,0.0328449770
AZN	Azerbaijani New Manat,0.7843996772,1.2748602900
BAM	Bosnian Convertible Marka,1.4168582378,0.7057869117
XAU	Gold Ounce,0.0007403859,1350.6469952403
IRR	Iranian Rial,24922.0357850581,0.0000401251
RSD	Serbian Dinar,82.6706941871,0.0120961849
LTL	Lithuanian Litas,2.5013053912,0.3997912464
BND	Bruneian Dollar,1.2365495679,0.8087019121
ETB	Ethiopian Birr,18.9690294553,0.0527175100
CRC	Costa Rican Colon,504.4756418455,0.0019822563
VEF	Venezuelan Bolivar,6.2877031954,0.1590405859
AFN	Afghan Afghani,56.2796950333,0.0177683976
TZS	Tanzanian Shilling,1603.4933517051,0.0006236384
UGX	Ugandan Shilling,2529.9992106967,0.0003952570
JMD	Jamaican Dollar,104.4247198928,0.0095762766
GEL	Georgian Lari,1.6679857654,0.5995255000
LVL	Latvian Lat,0.5089116734,1.9649775242
ZWD	Zimbabwean Dollar,361.9000000000,0.0027631943
BWP	Botswana Pula,8.3969989656,0.1190901659
CUC	Cuban Convertible Peso,1.0000000000,1.0000000000
ZMW	Zambian Kwacha,5.3099921292,0.1883241963
MMK	Burmese Kyat,971.9847174778,0.0010288228
GTQ	Guatemalan Quetzal,7.9825000655,0.1252740359
XCD	East Caribbean Dollar,2.7000000027,0.3703703700
LYD	Libyan Dinar,1.2254771997,0.8160086538
MKD	Macedonian Denar,44.5495980979,0.0224468916
TTD	Trinidadian Dollar,6.4387940540,0.1553085860
MZN	Mozambican Metical,29.6496304741,0.0337272332
ALL	Albanian Lek,101.8487586535,0.0098184800
BOB	Bolivian Boliviano,6.9099853785,0.1447181065
KZT	Kazakhstani Tenge,153.7249990391,0.0065051228
BBD	Barbadian or Bajan Dollar,2.0000000000,0.5000000000
AOA	Angolan Kwanza,97.5193039462,0.0102543800
KHR	Cambodian Riel,4059.9969796094,0.0002463056
XAG	Silver Ounce,0.0438137394,22.8238907132
AMD	Armenian Dram,405.5100708426,0.0024660300
UYU	Uruguayan Peso,21.5499939236,0.0464037254
MOP	Macau Pataca,7.9861730759,0.1252164197
NAD	Namibian Dollar,9.7653020351,0.1024033866
LBP	Lebanese Pound,1508.4400044415,0.0006629365
LAK	Lao or Laotian Kip,7877.3256471486,0.0001269466
BYR	Belarusian Ruble,9163.9851855612,0.0001091228
MGA	Malagasy Ariary	2189.1265017858,0.0004568032
SYP	Syrian Pound,112.9495533410,0.0088535100
VUV	Ni-Vanuatu Vatu,93.3747486119,0.0107095335
PGK	Papua New Guinean Kina,2.6315948754,0.3799977000
MNT	Mongolian Tughrik,1697.4874437773,0.0005891060
SDG	Sudanese Pound,4.4024828069,0.2271445554
ANG	Dutch Guilder,1.7730079874,0.5640132516
MWK	Malawian Kwacha,388.4160383224,0.0025745590
GMD	Gambian Dalasi,35.3157225597,0.0283160000
CUP	Cuban Peso,26.5000000000,0.0377358491
RWF	Rwandan Franc,668.1387948203,0.0014966950
MVR	Maldivian Rufiyaa,15.2655999083,0.0655067607
BTN	Bhutanese Ngultrum,61.4616075384,0.0162703196
SCR	Seychellois Rupee,12.0417771718,0.0830442206
HNL	Honduran Lempira,20.4247925504,0.0489601056
KPW	North Korean Won,127.3451647954,0.0078526735
PYG	Paraguayan Guarani,4456.1429428674,0.0002244093
DJF	Djiboutian Franc,179.4800497880,0.0055716499
XBT	Bitcoin,0.0048780488,204.9999999999
YER	Yemeni Rial,214.9001313143,0.0046533243
CDF	Congolese Franc,913.7940066307,0.0010943385
WST	Samoan Tala,2.2779043019,0.4390000050
GYD	Guyanese Dollar,202.9365497395,0.0049276486
AWG	Aruban or Dutch Guilder,1.7900000000,0.5586592179
MDL	Moldovan Leu,12.8649920082,0.0777303242
BZD	Belizean Dollar,1.9778058879,0.5056107913
HTG	Haitian Gourde,43.5501810932,0.0229620170
KGS	Kyrgyzstani Som,48.6899008381,0.0205381400
NIO	Nicaraguan Cordoba,24.9953986757,0.0400073635
CVE	Cape Verdean Escudo,80.4198052456,0.0124347478
KYD,Caymanian Dollar,0.8200001312,1.2195120000
GNF,Guinean Franc,6837.5012744265,0.0001462523
BSD,Bahamian Dollar,1.0000000000,1.0000000000
BIF,Burundian Franc,1549.9670173068,0.0006451750
SLL,Sierra Leonean Leone,4339.6773598933,0.0002304319
MRO,Mauritanian Ouguiya,298.4962890670,0.0033501254
TOP,Tongan Pa'anga,1.8066847387,0.5534999984
BMD,Bermudian Dollar,1.0000000000,1.0000000000
SBD,Solomon Islander Dollar,7.1710290427,0.1394500000
UZS,Uzbekistani Som,2168.9621516105,0.0004610500
SOS,Somali Shilling,1234.0245893256,0.0008103566
PAB,Panamanian Balboa,1.0000000000,1.0000000000
SRD,Surinamese Dollar,3.2748950531,0.3053532965
XDR,IMF Special Drawing Rights,0.6459499953,1.5481074500
SZL,Swazi Lilangeni,9.7653020351,0.1024033866
ERN,Eritrean Nakfa,10.4699971239,0.0955110100
LRD,Liberian Dollar,80.4980096299,0.0124226674
TJS,Tajikistani Somoni,4.7704946054,0.2096218700
TMT,Turkmenistani Manat,2.8500000000,0.3508771930
GIP,Gibraltar Pound,0.6175314631,1.6193506887
LSL,Basotho Loti,9.7653020351,0.1024033866
KMF,Comoran Franc,356.3952691699,0.0028058734
SVC,Salvadoran Colon,8.7500000000,0.1142857143
GGP,Guernsey Pound,0.6175314631,1.6193506887
XPT,Platinum Ounce,0.0006876848,1454.1546312300
STD,Sao Tomean Dobra,17780.3617591381,0.0000562418
IMP,Isle of Man Pound,0.6175314631,1.6193506887
FKP,Falkland Island Pound,0.6175314631,1.6193506887
XPD,Palladium Ounce,0.0013380914,747.3330867957
JEP,Jersey Pound,0.6175314631,1.6193506887
SHP,Saint Helenian Pound,0.6175314631,1.6193506887
SPL,Seborgan Luigino,0.1666666667,6.0000000000
TVD,Tuvaluan Dollar,1.0403201085,0.9612425943
