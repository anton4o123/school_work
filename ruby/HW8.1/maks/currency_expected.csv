MGA,Malagasy Ariary	2189.1265017858,0.0004568032,
XPT,Platinum Ounce,0.0006876848,1454.1546312300
XAU,Gold Ounce,0.0007403859,1350.6469952403
XPD,Palladium Ounce,0.0013380914,747.3330867957
XBT,Bitcoin,0.0048780488,204.9999999999
PKR,Pakistani Rupee	106.5198997274,0.0093879172,
XAG,Silver Ounce,0.0438137394,22.8238907132
SPL,Seborgan Luigino,0.1666666667,6.0000000000
KWD,Kuwaiti Dinar,0.2818497782,3.5479893100
BHD,Bahraini Dinar,0.3770993287,2.6518212151
OMR,Omani Rial,0.3850983907,2.5967389742
LVL,Latvian Lat,0.5089116734,1.9649775242
SHP,Saint Helenian Pound,0.6175314631,1.6193506887
GGP,Guernsey Pound,0.6175314631,1.6193506887
JEP,Jersey Pound,0.6175314631,1.6193506887
GIP,Gibraltar Pound,0.6175314631,1.6193506887
IMP,Isle of Man Pound,0.6175314631,1.6193506887
GBP,British Pound,0.6175314631,1.6193506887
FKP,Falkland Island Pound,0.6175314631,1.6193506887
XDR,IMF Special Drawing Rights,0.6459499953,1.5481074500
JOD,Jordanian Dinar,0.7082515975,1.4119276306
EUR,Euro,0.7244281138,1.3803992156
AZN,Azerbaijani New Manat,0.7843996772,1.2748602900
KYD,Caymanian Dollar,0.8200001312,1.2195120000
CHF,Swiss Franc,0.8920590797,1.1210019860
BMD,Bermudian Dollar,1.0000000000,1.0000000000
CUC,Cuban Convertible Peso,1.0000000000,1.0000000000
PAB,Panamanian Balboa,1.0000000000,1.0000000000
BSD,Bahamian Dollar,1.0000000000,1.0000000000
AUD,Australian Dollar,1.0403201085,0.9612425943
TVD,Tuvaluan Dollar,1.0403201085,0.9612425943
CAD,Canadian Dollar,1.0425283369,0.9592065410
